### PR TITLE
ISPN-6372 Remove deprecatd ReplicableCommand.get/SetParameters

### PR DESCRIPTION
**Pull request by pruivo**
https://issues.jboss.org/browse/ISPN-6372

### CODE REVIEW

**Comment by wburns** [comment]
Looks fine to me, just waiting for CI to finish.

**Comment by wburns** [comment]
Tests seem to be hung.  I am not sure if it is caused by this or just a test failure somewhere else.  Waiting for now.

File: core/src/main/java/org/infinispan/commands/module/ModuleCommandFactory.java [hunk 1]
```diff
@@ -36,19 +36,17 @@
     * id and argument array passed in.
```
Comment (wburns) [inline core/src/main/java/org/infinispan/commands/module/ModuleCommandFactory.java]: Need to update the javadoc here as well.

File: core/src/main/java/org/infinispan/commands/module/ModuleCommandFactory.java [hunk 2]
```diff
@@ -36,19 +36,17 @@
...
     * @param commandId command id to construct
-    * @param args array of arguments with which to initialize the ReplicableCommand
     * @return a ReplicableCommand
     */
-   ReplicableCommand fromStream(byte commandId, Object[] args);
+   ReplicableCommand fromStream(byte commandId);
 
    /**
     * Construct and initialize a {@link CacheRpcCommand} based on the command
     * id and argument array passed in.
```
Comment (wburns) [inline core/src/main/java/org/infinispan/commands/module/ModuleCommandFactory.java]: Need to update the javadoc here as well.

File: core/src/main/java/org/infinispan/commands/module/ModuleCommandFactory.java (hunk 1 above)
Comment (wburns) [inline core/src/main/java/org/infinispan/commands/module/ModuleCommandFactory.java]: Do we want to rename this method maybe even?

**Comment by wburns** [comment]
I doubt the test failures are caused by this, just minor comments.

**Comment by pruivo** [comment]
@wburns javadoc updated!

**Comment by wburns** [comment]
Pulling...

**Comment by wburns** [comment]
Integrated into master, thanks @pruivo !

**Pull request closed without merging**